generate.js

```diff
@@ -78,21 +74,11 @@ function generate (dir, log, cb) {
   })
 }
 
-const levels = {
-  'debug': 0,
-  'info': 1,
-  'error': 2
-}
-
-const colors = [
-  (l) => l,
-  chalk.green,
-  chalk.red
-]
-
 function cli (args) {
   const opts = argv(args)
 
+  log('info', 'Execute command: gen-app')
```

Comment:
I added it to each command due to Levenshtein disance. Its not always clear which command is executed when we have a common prefix like `gen`